recipes/pydevtool/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "0.2.0" %}
+
+package:
+  name: pydevtool
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pydevtool/pydevtool-{{ version }}.tar.gz
+  sha256: e821ce0b637b1bbf7638b1de23522b76c23b8e9900678ff46e6c846dcd9464e2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - doit
```

```suggestion
    - doit >=0.36.0
```
Ref: https://github.com/pydoit/pydevtool/blob/5556488ca8eb40d1368ef7fc1e46c8a843ccff6d/setup.cfg#L28

---

Ahh just saw that that was introduced after the release. All good.